src/sicmutils/calculus/coordinate.cljc

```diff
@@ -34,9 +34,11 @@
                     #(m/point->coords coordinate-system %)))
             (s/structure->access-chains prototype))))
 
-(defn ^:private quotify-coordinate-prototype
```

Comment:
Rather than use the `#'` var resolution trick to get past private, let's just make this public.

(This has to be available to SCI because the macroexpanded body of `let-coordinates` refers to this symbol.)